apps/dh/app-dh/src/app/datahub-app.component.ts

```diff
@@ -103,6 +103,11 @@ export class DataHubAppComponent implements OnInit {
       return '/';
     }
 
+    // Handle case where a new users is redirected to the app after sign up
+    if (path.startsWith('?code=')) {
```

Comment:
Consideration: Are we sure the `path` will always `startsWith` or should we use something like "includes" instead?